calculator.js

```diff
@@ -1,27 +1,90 @@
-
 const exampleAdditionInput = {
   num1: 3,
   num2: 5,
   operation: 'add',
 }
 
+function getValidOperator(operator) {
+  const operators = [ 'add', '+', 'subtract', '-', 'multiply', '*', 'divide', '/']
+
+  if (operators.includes(operator)){
+    return operator;
+  }
+  return undefined;
+}
+
+function getValidNums(number) {
+  if (isNaN) {
```

Comment:
This is just checking if there is a function named `isNan`.  You want to call it with `number`:

```suggestion
  if (!isNaN(number)) {
```

Reply:
It looks like you forked before the input tests were merged in though so no worries.  :smiley: